xcell/mappers/mapper_ACT_compsept.py

```diff
@@ -20,14 +20,13 @@ def __init__(self, config):
         self._get_ACT_defaults(config)
 
     def _get_signal_map(self):
-        if self.signal_map is None:
```

Comment:
this was a huge bug, right? We weren't really updating `self.signal_map`

Reply:
Did you update it? I think it didn't matter because if `self.signal_map` was `None`, `get_signal_map` return `self.signal_map` anyway without calling this function